hera_sim/tests/test_sigchain.py

```diff
@@ -182,5 +185,339 @@ def test_cross_coupling_xtalk(self):
         )
 
 
+class TestTimeVariation(unittest.TestCase):
```

Comment:
Don't use unittest stuff! Use `pytest` fixtures / parameterizations

Reply:
Gotcha. I'll rewrite the new tests to use `pytest` only, and make the tests more atomic.

Reply:
All of the tests for this module (including the old ones) have been updated to use `pytest` only. Some further consolidation can be done, and some tests may be able to be compactified by exploiting parameterization, but I'd like to leave that for the PR where I convert *all* of the tests to `pytest` only.

Reply:
Excellent!